cpp/include/kvikio/file_handle.hpp

```diff
@@ -496,6 +496,79 @@ class FileHandle {
     return parallel_io(op, devPtr_base, size, file_offset, task_size, devPtr_offset);
   }
 
+  /**
+   * @brief Reads specified bytes from the file into the device memory.
+   *
+   * This API reads asynchronously the data from the GPU memory to the file at a specified offset
+   * and size bytes by using GDS functionality. The API works correctly for unaligned
```

Comment:
This sentence is in contradiction with the rest of the docstring? I think you mean:

> This API reads size bytes asynchronously from the file into device memory writing to a specified offset using GDS functionality.

?